virt/virt-2-6-release-notes.adoc

```diff
@@ -32,10 +32,28 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-7544: Microsoft's Windows Server Virtualization Validation Program (SVVP) applies to RHCOS workers
 
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS 8 workers. In the Microsoft SVVP Catalog, they are named Red Hat OpenShift Container Platform 4 on RHEL CoreOS 8.
```

Comment:
It's just CoreOS, not CoreOS 8. I do not think that we make a differentiation anywhere else inthe docs, do we?

Reply:
Changed to only "CoreOS"